Formula/ghostscript.rb

```diff
@@ -47,25 +55,32 @@ class Ghostscript < Formula
     sha256 "0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401"
   end
 
-  # Fix build on ARM Big Sur, updating config.{guess,sub}
-  # https://bugs.ghostscript.com/show_bug.cgi?id=703095
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/98c39e09/ghostscript/config.patch"
-    sha256 "155cf2ee5a498d441c3194ba3d75cb7812beaa3f507a72017174a884bf742862"
-  end
-
   def install
-    on_linux do
-      # Fixes: ./soobj/dxmainc.o: file not recognized: File truncated
-      ENV.deparallelize
-    end
+    # Fix vendored tesseract build error: 'cstring' file not found
+    # Remove when possible to link to system tesseract
+    ENV.append_to_cflags "-stdlib=libc++" if ENV.compiler == :clang
+
+    # The VERSION file is incorrectly included as C++20 <version> header
+    # Remove when upstream fixes build issue on case-insensitive filesystems
+    rm "tesseract/VERSION"
+
+    # Delete local vendored sources so build uses system dependencies
+    rm_rf "expat"
+    rm_rf "freetype"
+    rm_rf "jbig2dec"
+    rm_rf "jpeg"
+    rm_rf "lcms2mt"
+    rm_rf "libpng"
+    rm_rf "openjpeg"
+    rm_rf "zlib"
 
     args = %W[
       --prefix=#{prefix}
-      --disable-cups
       --disable-compile-inits
-      --disable-gtk
+      --disable-cups
       --disable-fontconfig
```

Comment:
For Linux, it seems odd that the disable flags are set but dependencies (fontconfig, libidn) are added...

Reply:
Does seem weird. Not sure we need to disable `cups` either, as that is provided by macOS. Unless we need it to avoid build failure.